modules/ssh/ssh.go

```diff
@@ -124,6 +176,14 @@ func runSSHCommand(sshSession *SshSession) (string, error) {
 		return "", err
 	}
 
+	if sshSession.Input != nil {
+		w, err := sshSession.Session.StdinPipe()
+		if err != nil {
+			return "", err
+		}
+		go (*sshSession.Input)(w)
```

Comment:
Does anything need to be closed (in a `defer`) after this?

Reply:
Refactored the code to make it more obvious where the closing was being handled